README.md

```diff
@@ -54,7 +55,7 @@ Stream text with GPT-4o, transcribe and translate audio with Whisper, or create
       - [Translate](#translate)
       - [Transcribe](#transcribe)
       - [Speech](#speech)
-    - [Errors](#errors)
+    - [Errors](#errors-1)
```

Comment:
In this case, the link was pointing to the "Errors" heading above, under "Logging." `errors-1` matches the ID for this section.